grails-app/controllers/org/pih/warehouse/api/OrganizationApiController.groovy

```diff
@@ -24,4 +25,10 @@ class OrganizationApiController extends BaseDomainApiController {
         render ([data:organizations] as JSON)
      }
 
+    def listSuppliers = {
```

Comment:
Now that I've had a few more minutes to look at this, why don't we just move roleType to the URL in option-utils.jsx instead of creating a new endpoint.